test/Worker.Extensions.DurableTask.Analyzers.Test/Verifiers/CSharpAnalyzerVerifier`1.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.CodeAnalysis;
```

Comment:
adding copyright notice :)

```suggestion
﻿// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.

﻿using Microsoft.CodeAnalysis;
```